docs/ProductSummaryList.md

```diff
@@ -27,29 +27,42 @@ This block is used to specify what variation of `product-summary` to be used to
 
 For `SkusFilterEnum`:
 
-| Name | Value | Description |
-| ---- | ----- | ----------- |
+| Name            | Value             | Description                                                                                                                                            |
+| --------------- | ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | First Available | `FIRST_AVAILABLE` | Most performant, ideal if you do not have a SKU selector in your shelf. Will return only the first available SKU for that product in your shelf query. |
-| All Available | `ALL_AVAILABLE` | A bit better performace, will only return SKUs that are available, ideal if you have a SKU selector but still want a better performance. |
-| All | `ALL` | Returns all SKUs related to that product, least performant option. |
+| All Available   | `ALL_AVAILABLE`   | A bit better performace, will only return SKUs that are available, ideal if you have a SKU selector but still want a better performance.               |
+| All             | `ALL`             | Returns all SKUs related to that product, least performant option.                                                                                     |
 
 For `InstallmentCriteriaEnum`:
 
-| Name | Value | Description |
-| ---- | ----- | ----------- |
-| Maximum without interest | `MAX_WITHOUT_INTEREST` | Will display the maximum installment option with no interest. |
-| Maximum | `MAX_WITH_INTEREST` | Will display the maximum installment option having interest or not. |
+| Name                     | Value                  | Description                                                         |
+| ------------------------ | ---------------------- | ------------------------------------------------------------------- |
+| Maximum without interest | `MAX_WITHOUT_INTEREST` | Will display the maximum installment option with no interest.       |
+| Maximum                  | `MAX_WITH_INTEREST`    | Will display the maximum installment option having interest or not. |
+
+For `PreferredSKUEnum`:
+
+| Name            | Value             | Description                                        |
+| --------------- | ----------------- | -------------------------------------------------- |
+| First Available | `FIRST_AVAILABLE` | Selects the first available SKU in stock it finds. |
+| Last Available  | `LAST_AVAILABLE`  | Selects the last available SKU in stock it finds.  |
+| Cheapest        | `PRICE_ASC`       | Selects the cheapest SKU in stock it finds.        |
+| Most Expensive  | `PRICE_DESC`      | Selects the most expensive SKU in stock it finds.  |
+
+⚠️ Keep in mind that the default behaviour for the initially selected SKU, has now changed to a Product (field) specification.  
```

Comment:
I don't quite understand what you mean here. The default component behavior has changed?

Reply:
Yes, previously the initially selected SKU was the first available, now it's the product specification

Reply:
But wouldn't this affect all stores then? If so this would be a breaking change